packages/rancher-alerting-drivers/charts/Chart.yaml

```diff
@@ -12,9 +12,10 @@ dependencies:
   name: sachet
   repository: file://./charts/sachet
 description: The manager for third-party webhook receivers used in Prometheus Alertmanager
+icon: https://raw.githubusercontent.com/rancher/charts/dev-v2.6/assets/logos/alerting-drivers.svg
 keywords:
 - monitoring
 - alertmanger
 - webhook
 name: rancher-alerting-drivers
-version: 1.0.2
+version: 1.1.0
```

Comment:
No need to modify for this PR, but in future I think it's good to have one commit per version bump (e.g. the equivalent of the base copy commit in rancher/system-charts) and then one commit with your changes.